integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -51,6 +51,11 @@ setup() {
 	docker run --rm --runtime=$RUNTIME $image cat /etc/alpine-release
 }
 
+@test "[run application] run an amazonlinux container" {
+	image="amazonlinux"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "amazon-linux-extras install -y firecracker"
```

Comment:
and run `firecracker --help` to make sure it was installed .. ?

Reply:
@devimc changes applied